src/index.ts

```diff
@@ -977,27 +977,55 @@ export async function getTotalDiskCapacity() {
   return -1;
 }
 
+export async function getTotalDiskCapacityNew() {
+  if (Platform.OS === 'android' || Platform.OS === 'ios' || Platform.OS === 'web') {
+    return RNDeviceInfo.getTotalDiskCapacityNew();
+  }
+  return -1;
+}
+
 export function getTotalDiskCapacitySync() {
   if (Platform.OS === 'android' || Platform.OS === 'ios' || Platform.OS === 'web') {
     return RNDeviceInfo.getTotalDiskCapacitySync();
   }
   return -1;
 }
 
+export function getTotalDiskCapacityNewSync() {
+  if (Platform.OS === 'android' || Platform.OS === 'ios' || Platform.OS === 'web') {
+    return RNDeviceInfo.getTotalDiskCapacityNewSync();
+  }
+  return -1;
+}
+
 export async function getFreeDiskStorage() {
   if (Platform.OS === 'android' || Platform.OS === 'ios' || Platform.OS === 'web') {
     return RNDeviceInfo.getFreeDiskStorage();
   }
   return -1;
 }
 
+export async function getFreeDiskStorageNew() {
+  if (Platform.OS === 'android' || Platform.OS === 'ios' || Platform.OS === 'web') {
+    return RNDeviceInfo.getFreeDiskStorageNew();
+  }
+  return -1;
+}
+
 export function getFreeDiskStorageSync() {
   if (Platform.OS === 'android' || Platform.OS === 'ios' || Platform.OS === 'web') {
     return RNDeviceInfo.getFreeDiskStorageSync();
   }
   return -1;
 }
 
+export function getFreeDiskStorageNewSync() {
+  if (Platform.OS === 'android' || Platform.OS === 'ios' || Platform.OS === 'web') {
+    return RNDeviceInfo.getFreeDiskStorageNewSync();
```

Comment:
this is the last of this block of 4 but the same question ("will it crash on ios and web since those did not get a new native API?") apply to all 4